packages/functional-tests/pages/settings/components/unitRow.ts

```diff
@@ -39,31 +39,36 @@ export class UnitRow {
 }
 
 export class AvatarRow extends UnitRow {
-  async isDefault() {
-    const el = await this.page.$('[data-testid=avatar-nondefault]');
-    if (!el) {
-      return true;
-    }
-    const src = await el.getAttribute('src');
-    return src?.includes('/avatar/');
+  get photo() {
+    return this.page
+      .getByTestId('settings-profile')
+      .getByTestId('avatar-nondefault');
   }
 
-  clickAdd() {
-    return this.clickCta();
+  get addButton() {
```

Comment:
Even though they have the same locator, I would like to keep the distinct buttons, because it makes the tests more readable/easier to understand.

I was genuinely confused the first time reading through `avatar.spec.ts > upload and remove avatar profile photo` when we were clicking to add an avatar even though in the step before we already added the avatar and the UI has a `change` button.